scripts/write_notes_to_couchdb.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+DESC="""Common script called by other EPP scripts to write notes to couchdb
+"""
+import json
+import yaml
+import couchdb
+import smtplib
+import os
+from email.mime.text import MIMEText
+
+
+def write_note_to_couch(pid, timestamp, note, lims):
+    configf = '~/.statusdb_cred.yaml'
+    with open(os.path.expanduser(configf)) as config_file:
+        config = yaml.safe_load(config_file)
+    if not config['statusdb']:
+        print('Statusdb credentials not found')
```

Comment:
Maybe this should be an email error as well?

Reply:
Done!